packages/wrangler/src/index.ts

```diff
@@ -531,8 +531,8 @@ export function createCLIParser(argv: string[]) {
 	});
 
 	// r2
-	wrangler.command("r2", "📦 Manage R2 buckets & objects", (r2Yargs) => {
-		return r2(r2Yargs.command(subHelp));
+	wrangler.command("r2", "📦 Interact with an R2 store", (r2Yargs) => {
```

Comment:
```suggestion
	wrangler.command("r2", "📦 Manage R2 buckets & objects", (r2Yargs) => {
```

Reply:
wording was changed recently based on a request from the design team, so let's please keep it in place. Similarly, wording across multiple other commands was changed in the same way to bring consistency across the cli

Reply:
Ooh, good catch. Not entirely sure why I changed it back, but will change it back again...